tests/torchtune/datasets/test_instruct_dataset.py

```diff
@@ -4,146 +4,91 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from unittest import mock
+from pathlib import Path
 
-from datasets import Dataset
+import pytest
 from tests.test_utils import DummyTokenizer
 from torchtune.data import InstructTemplate
 from torchtune.data._common import CROSS_ENTROPY_IGNORE_IDX
-from torchtune.datasets import InstructDataset
+from torchtune.datasets import instruct_dataset, InstructDataset
+
+ASSETS = Path(__file__).parent.parent.parent / "assets"
```

Comment:
Not a new problem to this PR but why do we define this in every test file instead of just importing from test_utils or something?